recipes/toml11/all/test_package/test_package.cpp

```diff
@@ -1,4 +1,4 @@
-#include <toml11/toml.hpp>
```

Comment:
Yes, no namespace:
https://github.com/ToruNiina/toml11/blob/3c3ebd88b4d73d220eea18a2a00fcd93782e8611/CMakeLists.txt#L51
https://github.com/ToruNiina/toml11/blob/3c3ebd88b4d73d220eea18a2a00fcd93782e8611/CMakeLists.txt#L76
It might conflict with others toml libs, I don't know, but it's the official way to include toml11 according to CMakeLists